site/content/docs/integrations/prometheus-v2.md

```diff
@@ -42,6 +41,10 @@ Here is an example
 
 Now restart Prometheus and you should see metrics coming in.
 
+{{< warning >}}
+The Prometheus metrics endpoint has a rate limit of 8 requests per 5 minutes. We recommend using a scrape interval of 60 seconds.
```

Comment:
I'm not so sure about mentioning the exact rate limit.

It's 8 requests per 5 minutes (rather than 1 request per 45 seconds) so that there's a buffer to run some `curl` requests for debugging.

Reply:
let's leave it at this for now.